src/containers/AccountConnection.jsx

```diff
@@ -16,6 +16,8 @@ class AccountConnection extends Component {
     this.state = {
       account: this.props.existingAccount
     }
+
+    if (this.props.error) this.handleError({message: this.props.error})
```

Comment:
It makes me think that I want to specify in cozy-guidelines that we should always provide error the same way, i.e. with a native Error object, or at least an object the nearest as possible of an Error object. It means pretty mush only a `message` attribute.
In this case, it could allow us to do only :
```js
if (this.props.error) this.handleError(this.props.error)
```